src/__tests__/vitest.setup.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { createSerializer } from '@emotion/jest';
```

Comment:
Emotion støtter egentlig ikke vitest, men vitest og jest har samme oppsett for serializers.